infra/cifuzz/fuzz_target.py

```diff
@@ -30,9 +33,15 @@
 
 LIBFUZZER_OPTIONS = '-seed=1337 -len_control=0'
 
+# Location of google cloud storage for latest OSS-Fuzz builds.
+GCS_BASE_URL = 'https://storage.googleapis.com/clusterfuzz-builds'
+
 # The number of reproduce attempts for a crash.
 REPRODUCE_ATTEMPTS = 10
 
+# The name to store the latest OSS-Fuzz build at.
+BUILD_STORE_NAME = 'oss_fuzz_latest.zip'
```

Comment:
nit: IMO `BUILD_ARCHIVE_NAME` would be more explicit and better, you download, unpack, and delete it, while the actual build is still stored somewhere else

Reply:
Done.